modules/microshift-low-latency-rhel-edge-blueprint-rtk.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-low-latency-blueprint-rhel-edge-rtk_{context}"]
+= Reference blueprint for installing {op-system-rt-kernel} in a {op-system-ostree} image
+
+An image blueprint is a persistent definition of the required image customizations that enable you to create multiple builds. Instead of reconfiguring the blueprint for each image build, you can edit, rebuild, delete, and save the blueprint so that you can keep rebuilding images from it.
+
+* A blueprint groups the images that were built from it. This means that you can have easier access to related images, especially when you are dealing with lots of images.
+
+* Customizations stored in a blueprint are always applied across all builds resulting from the blueprint. This is true even if the resulting builds, and images from those builds, have different content versions.
+
+* You can create a blueprint with a name, specify the packages that you want to install, and define other customizations, such as including {microshift-short} and low-latency settings.
+
+* Images created from a blueprint take the name of the blueprint unless you define a name.
+
+.Example blueprint used to install the real-time kernel in a {op-system-ostree} image
+[source,text]
+----
+name = "microshift-low-latency"
+description = "RHEL 9.4 and MicroShift configured for low latency"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
+
+[[packages]]
+name = "microshift"
+version = "*"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "*"
+
+[[packages]]
+name = "microshift-networking"
+version = "*"
+
+[[packages]]
+name = "microshift-selinux"
+version = "*"
+
+[[packages]]
+name = "microshift-low-latency"
+version = "*"
+
+[[packages]]
+name = "systemd-resolved"
+version = "*"
```

Comment:
```suggestion
[[packages]]
name = "systemd-resolved"
version = "*"
```

I think we should remove it because it's only needed for some tests in CI and I just copied the whole thing.

Reply:
Gotcha. I thought this was always there as a RHEL package.